ci/bazel-scripts/targets.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python3
+#
+#   targets.py [-h] [--skip_long_tests] [--commit_range COMMIT_RANGE] {build,test}
+#
+# This script is invoked from the CI Main workflow to print to stdout which Bazel targets should be built or tested.
```

Comment:
nit: 

> from the CI Main workflow

subject to change, might get out of date

Reply:
I like to give the reader a sense of where a script fits in but I can also see value in having it be completely independent so I removed the reference.